package.json

```diff
@@ -30,14 +30,14 @@
     "date-fns": "^2.28.0"
   },
   "devDependencies": {
-    "@appuniversum/ember-appuniversum": "^1.2.1",
+    "@appuniversum/ember-appuniversum": "^1.10.0",
     "@ember/legacy-built-in-components": "^0.4.1",
     "@ember/optional-features": "^2.0.0",
     "@ember/test-helpers": "^2.6.0",
     "@glimmer/component": "^1.0.4",
     "@glimmer/tracking": "^1.0.4",
     "@lblod/ember-acmidm-login": "^1.0.0",
-    "@lblod/ember-jobs-dashboard-engine": "^0.4.0",
+    "@lblod/ember-jobs-dashboard-engine": "^0.5.0",
```

Comment:
The engine also had some deprecation warnings, so we had to do some maintenance over there as well.